Python/cim-convert-tool/configOpCim17.py

```diff
@@ -0,0 +1,182 @@
+configOpCim17 = \
+    {
+        'cim:Analog': {
+            'mainAttributes': {
+                'rdf:ID': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:IdentifiedObject.mRID': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.aliasName': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.description': {'type': 'string', 'list': False},
+                'cim:IdentifiedObject.name': {'type': 'string', 'list': False},
+                # 'cim:IdentifiedObject.shortName': {'type': 'string', 'list': False},
+                'cim:Measurement.measurementType': {'type': 'string', 'list': False},
+                'cim:Analog.maxValue': {'type': 'float', 'list': False},
+                'cim:Analog.minValue': {'type': 'float', 'list': False},
+                'cim:Analog.normalValue': {'type': 'float', 'list': False},
+                'cim:Analog.positiveFlowIn': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+                'cim:Measurement.phases': {'rdf:resource': {'type': 'string'}, 'list': False},
+                'cim:Measurement.unitMultiplier': {'rdf:resource': {'type': 'string'}, 'list': False},
+                'cim:Measurement.unitSymbol': {'rdf:resource': {'type': 'string'}, 'list': False},
+                'cim:Measurement.Asset': {'rdf:resource': {'type': 'string'}, 'list': False},
+                'cim:Measurement.PowerSystemResources': {'rdf:resource': {'type': 'string'}, 'list': False}
+            }
+        },
+        'cim:AnalogValue': {
```

Comment:
AnalogValue missing reference MeasurementValue.MeasurementValueSource and MeasurementValue.RemoteSource